docs/reference/plugins/py-terminal.md

```diff
@@ -0,0 +1,27 @@
+# &lt;py-terminal&gt;
+
+This is one of the core plugins in PyScript, which is active by default. With it, you can receive `stdout` and `stderr` from your python code, which will be displayed on the page in `<py-terminal>`.
+
+## Configuration
+
+You can control how `<py-terminal>` behaves by setting the value of the  `terminal` configuration in your `<py-config>`.
+
+| value | description |
+|-------|-------------|
+| `false` | Deactivate the plugin, no `<py-terminal>` will be added to the page |
```

Comment:
```suggestion
| `false` | Don't add `<py-terminal>` to the page |
```

the plugin is still "active", in the sense that you can still place a `<py-terminal>` manually and it works.